### PR TITLE
Modify Admin Router log format

### PR DESCRIPTION
**Pull request by jongiddy**
## High-level description

This format provides a clearer layout for Admin Router logs:
- AR worker
- `<` client identity and request
- `>` proxy server identity and response, with `code=xxx` to make searching for response statuses easier

It also adds the connection ID which is useful for matching access logs to error logs, and the source port which can be useful for matching with client logs.

With these changes, logs now look like:
```
Jan 10 14:53:00 master1 nginx[3632]: master1 nginx: 2020/01/10 14:53:00 [notice] 3632#0: *77 [lua] common.lua:183: validate_jwt(): UID from the valid DC/OS authentication token: `dcos_checks_master`, client: 172.17.0.2, server: master.mesos, request: "GET /exhibitor/exhibitor/v1/ui/index.html HTTP/1.1", host: "172.17.0.2:443"
Jan 10 14:53:00 master1 nginx[3632]: master1 nginx: 2020/01/10 14:53:00 [notice] 3632#0: *77 [lua] ee.lua:201: check_access_control_entry_or_exit(): Consult policyquery via `/acs/api/v1/internal/policyquery?rid=dcos:adminrouter:ops:exhibitor&uid=dcos_checks_master&action=full`, client: 172.17.0.2, server: master.mesos, request: "GET /exhibitor/exhibitor/v1/ui/index.html HTTP/1.1", host: "172.17.0.2:443"
Jan 10 14:53:00 master1 nginx[3632]: master1 nginx: 2020/01/10 14:53:00 [notice] 3632#0: *77 [lua] ee.lua:60: auditlog(): type=audit timestamp=2020-01-10T14:53:00Z authorizer=adminrouter object=dcos:adminrouter:ops:exhibitor action=full result=allow reason="IAM PQ response" srcip=172.17.0.2 srcport=41066 request_uri=/exhibitor/exhibitor/v1/ui/index.html uid=dcos_checks_master, client: 172.17.0.2, server: master.mesos, request: "GET /exhibitor/exhibitor/v1/ui/index.html HTTP/1.1", host: "172.17.0.2:443"
Jan 10 14:53:00 master1 nginx[3632]: master1 nginx: 10/Jan/2020:14:53:00 +0000 3632 *77 <172.17.0.2:41066 Go-http-client/1.1 - GET /exhibitor/exhibitor/v1/ui/index.html HTTP/1.1 len=594 >127.0.0.1:8181 code=200 len=28983(28831) 0.000 0.008-0.008 0.025
Jan 10 14:53:00 master1 nginx[3632]: master1 nginx: 2020/01/10 14:53:00 [info] 3632#0: *77 client 172.17.0.2 closed keepalive connection
```

All of these lines are part of the nginx error_log, except the second-last, which is part of the access_log. The `*77` ties these lines together so that we can see they are the same request.

The access log line (after the journald prefix) is 200 characters, the previous equivalent was 292 characters. This is significant in that we also have customer requests to reduce Admin Router log sizes.

## Corresponding DC/OS tickets (required)

  - [DCOS-59598](https://jira.mesosphere.com/browse/DCOS-59598) Add connection id to Admin Router log lines.

## Related tickets (optional)

  - [DCOS-ID](https://jira.mesosphere.com/browse/DCOS-<number>) JIRA title / short description.

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain.
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

**Comment by jongiddy** [comment]
@mesosphere-mergebot label Ready For Review

File: packages/adminrouter/extra/src/includes/http/common-windows.conf [hunk 1]
```diff
@@ -86,19 +86,12 @@
...
-    '"$http_referer" "$http_user_agent" '
-    'logtime_msec=$msec '
-    'upstream_addr=$upstream_addr '
-    'request_time=$request_time '
-    'upstream_response_time=$upstream_response_time '
-    'upstream_connect_time=$upstream_connect_time '
-    'upstream_header_time=$upstream_header_time';
+# Define custom log format.
+log_format customformat '$time_local $pid *$connection '
+    '<$remote_addr:$remote_port $http_user_agent $http_referer '
```
Comment (orsenthil) [inline packages/adminrouter/extra/src/includes/http/common-windows.conf]: Just a note: I was confused by `<` and `>` in this and assumed they were special characters.
* It seems to me that we want to indicate the log-line in this format `< some-log-line >`  - Is this correct? Will it help if we had both start-symbol `<` and and end-symbol `>` in the same line?

**Comment by jongiddy** [comment]
The <> are not delimiters, the < points to the left where you might draw
the client and the > points to the right where you might draw the upstream
service.

**Comment by orsenthil** [comment]
Thanks for the explanation, @jongiddy  

I could not find the relevant documentation initially, but I found them now

http://httpd.apache.org/docs/current/mod/mod_log_config.html

```
The modifiers "<" and ">" can be used for requests that have been internally redirected to choose whether the original or final (respectively) request should be consult
```

**Pull request merged**